neo/bin/export.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+from neo.Settings import settings
+from neo.Implementations.Blockchains.LevelDB.LevelDBBlockchain import LevelDBBlockchain
+from neo.Core.Blockchain import Blockchain
+import argparse
+from tqdm import trange
+import binascii
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-m", "--mainnet", action="store_true", default=False,
+                        help="use MainNet instead of the default TestNet")
+    parser.add_argument("-c", "--config", action="store", help="Use a specific config file")
+
+    # Where to store stuff
+    parser.add_argument("--datadir", action="store",
+                        help="Absolute path to use for database directories")
+
+    parser.add_argument("-o", "--output", help="Where to save output file")
+
+    parser.add_argument("-t", "--totalblocks", help="Total blocks to export", type=int)
+
+    args = parser.parse_args()
+
+    if args.mainnet and args.config:
+        print("Cannot use both --config and --mainnet parameters, please use only one.")
+        exit(1)
+
+    # Setting the datadir must come before setting the network, else the wrong path is checked at net setup.
+    if args.datadir:
+        settings.set_data_dir(args.datadir)
+
+    # Setup depending on command line arguments. By default, the testnet settings are already loaded.
+    if args.config:
+        settings.setup(args.config)
+    elif args.mainnet:
+        settings.setup_mainnet()
```

Comment:
Settings are not using testnet by default anymore, so I'd recommend adding `else: settings.setup_testnet()`